### DISCUSSION TITLE
Cleanup PHP namespaces

### DISCUSSION THREAD

**Discussion by shyim** [shopware/platform#3168] [RFC / Proposal]
**Effort**: middle

### Description

Inside our domains we have some naming conflicts or "bundles" which are in the wrong domain. With this proposal, we want to define a rule how files are structured in the core and apply it

### Benefits

* Unify code/domains 
* Define strict conditions for all naming

**Comment by lasomethingsomething** [comment]
@shyim Something we still want/need to do or can we close?

**Reply by keulinho** [reply]
@lasomethingsomething already closed a year ago
![Screenshot 2025-03-14 at 12 40 32](https://github.com/user-attachments/assets/16079da0-8b44-4147-833c-30c9f0dc3c9d)
